### PR TITLE
Updating test class names per Mike Smith

### PR DESCRIPTION
**Pull request by ptewson-sfdo**
# Critical Changes

# Changes

# Issues Closed

# New Metadata

# Deleted Metadata

### CODE REVIEW

File: src/classes/BDI_FieldMappingCustomMetadataTest.cls [hunk 1]
```diff
@@ -36,57 +36,57 @@
 */
 @isTest
 private class BDI_FieldMappingCustomMetadataTest {
```
Comment (psadie) [inline src/classes/BDI_FieldMappingCustomMetadataTest.cls]: Looks like this one wasn't updated

File: src/package.xml [hunk 2]
```diff
@@ -59,6 +59,7 @@
         <members>BDE_BatchEntry_TEST</members>
         <members>BDI_AdditionalObjectService</members>
         <members>BDI_AdditionalObjectServiceTest</members>
```
Comment (psadie) [inline src/package.xml]: No longer needed?

File: src/package.xml [hunk 3]
```diff
@@ -85,16 +86,20 @@
         <members>BDI_FieldMapping</members>
         <members>BDI_FieldMappingCustomMetadata</members>
         <members>BDI_FieldMappingCustomMetadataTest</members>
```
Comment (psadie) [inline src/package.xml]: Here also

File: src/package.xml [hunk 4]
```diff
@@ -85,16 +86,20 @@
...
+        <members>BDI_FieldMappingCustomMetadata_TEST</members>
         <members>BDI_FieldMappingHelpText</members>
         <members>BDI_IMatchDonations</members>
         <members>BDI_IPostProcess</members>
         <members>BDI_ManageAdvancedMappingCtrl</members>
         <members>BDI_ManageAdvancedMappingCtrlTest</members>
```
Comment (psadie) [inline src/package.xml]: Here also

File: src/package.xml [hunk 5]
```diff
@@ -85,16 +86,20 @@
...
+        <members>BDI_ManageAdvancedMappingCtrl_TEST</members>
         <members>BDI_MatchDonations</members>
         <members>BDI_MigrationMappingHelper</members>
         <members>BDI_MigrationMappingHelperTest</members>
```
Comment (psadie) [inline src/package.xml]: Here also

File: src/package.xml [hunk 6]
```diff
@@ -85,16 +86,20 @@
...
+        <members>BDI_MigrationMappingHelper_TEST</members>
         <members>BDI_MigrationMappingUtility</members>
         <members>BDI_MigrationMappingUtilityTest</members>
```
Comment (psadie) [inline src/package.xml]: Here also

File: src/package.xml [hunk 7]
```diff
@@ -2691,6 +2691,34 @@
         <members>Trigger_Handler__c-iw</members>
         <members>Trigger_Handler__c-ja</members>
         <members>Trigger_Handler__c-nl_NL</members>
+        <members>Data_Import_Field_Mapping_Set__mdt-de</members>
```
Comment (force2b) [inline src/package.xml]: @patrick-yan-sf  Were these not in the package.xml in the base PR?

Comment (kenlewis-sfdo) [reply src/package.xml]: @ptewson-sfdo

Comment (ptewson-sfdo) [reply src/package.xml]: Nope - I apparently missed that step, although I thought they had been in there. They are there here now!

**Pull request merged**